### ISSUE TITLE
Settings: Description text color contrast

### ISSUE THREAD

**Issue by ashleighaxios** [Automattic/wp-calypso#12173]
Some of the small description text, usually italicized, isn't picking up the new Calypso style adjustment for color contrast.

The style that's correct & should be cascading down: gray-text-min
(minimum light value)

Places where it's not appearing correctly:
- General > Jetpack card > "Last synced X days"
- Writing > Categories > Category 'tags' & Tags 'tags'

**Comment by ashleighaxios** [comment]
Cc @rickybanister @MichaelArestad

**Comment by tyxla** [comment]
This issue is being addressed in #12223, so I'm unassigning Rocco and Oscar.
